apps/api/src/services/meme.service.ts

```
@@ -0,0 +1,83 @@
+      // Update with incremented count
+      const document = await databases.updateDocument(DATABASE_ID, memeCollectionId, memeId, {
+        usageCount: currentCount + 1,
+      });
+
+      return formatDocument(document);
+    } catch (error) {
+      console.error(`Error incrementing usage count for meme ${memeId}:`, error);
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)